src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -1,6 +1,6 @@
 <dashboard-app class="app-container dashboard-container">
   <div class="fullScreenModePlaceholder">
-    <button
+    <div
```

Comment:
👍  Good thinking @snide. `kbn-accessible-click` adds those properties under the hood, so good call @Stacey-Gammon.